cmd/api/src/analysis/ad/ntlm_integration_test.go

```diff
@@ -102,16 +103,29 @@ func TestPostNTLMRelaySMB(t *testing.T) {
 		groupExpansions, computers, domains, authenticatedUsers, err := fetchNTLMPrereqs(db)
 		require.NoError(t, err)
 
+		ldapSigningCache, err := ad2.FetchLDAPSigningCache(testContext.Context(), db)
+		require.NoError(t, err)
+
+		protectedUsersCache, err := ad2.FetchProtectedUsersMappedToDomains(testContext.Context(), db, groupExpansions)
+		require.NoError(t, err)
+
 		for _, domain := range domains {
 			innerDomain := domain
 
 			err = operation.Operation.SubmitReader(func(ctx context.Context, tx graph.Transaction, outC chan<- analysis.CreatePostRelationshipJob) error {
 				for _, computer := range computers {
 					innerComputer := computer
 					domainSid, _ := innerDomain.Properties.Get(ad.DomainSID.String()).String()
-					authenticatedUserID := authenticatedUsers[domainSid]
 
-					if err = ad2.PostCoerceAndRelayNTLMToSMB(tx, outC, groupExpansions, innerComputer, authenticatedUserID); err != nil {
+					if authenticatedUserID, ok := authenticatedUsers[domainSid]; !ok {
```

Comment:
The requirements for e NTLMToSMB node changed to include the need for the `functionallevel` and Protected Users groups. So the test needed to be adjusted to accomodate